src/io/flutter/actions/FlutterCleanAction.java

```diff
@@ -14,7 +15,7 @@
 
 public class FlutterCleanAction extends FlutterSdkAction {
   @Override
-  public void startCommand(@NotNull Project project, @NotNull FlutterSdk sdk, @Nullable PubRoot root) {
+  public void startCommand(@NotNull Project project, @NotNull FlutterSdk sdk, @Nullable PubRoot root, @NotNull DataContext context) {
```

Comment:
What is the DataContext used for?

Reply:
I traced it down to ExecutionEnvironment. I stopped tracing when I got to a platform class, so I don't have a good answer for that.

Reply:
But why are we adding it as a parameter here?

Reply:
OK, it looks like it's used in the startCommand() method of the `AttachDebuggerAction` class.